app/views/activities/_form.html.haml

```diff
@@ -1,39 +1,27 @@
-= simple_form_for @activity, :html => { :class => 'form-horizontal' } do |f|
-  = f.association :category, label: "What", required: false
-  = f.input :date, required: false, as: :string
-  - if @activity.persisted?
-    = f.input :description, as: :string, required: false
-  - else
-    .descriptions
-      = f.input :description, as: :string, required: false, input_html: { name: 'activity[descriptions][]' }
-    = link_to " add description", 'javascript:', class: 'icon-plus controls'
-
-  .form-actions
-    = f.button :submit, :class => 'btn-primary'
-    = link_to t('.cancel', :default => t("helpers.links.cancel")), activities_path, :class => 'btn'
+.col-md-4
+  = simple_form_for @activity, html: { class: 'form-horizontal' } do |f|
+    = f.association :category, label: "What", required: false, input_html: { class: 'form-control' }
+    = f.input :date, required: false, as: :string, input_html: { class: 'form-control' }
     - if @activity.persisted?
-      = link_to 'Delete', activity_path(@activity), method: :delete, class: 'btn btn-danger', confirm: "Are you sure you want to delete entry?"
-
-.template{style: "display: none"}
-  .control-group.string.optional
-    %label.string.optional.control-label{:for => "activity_description"} Description
-    .controls
-      %input#activity_description.string.optional{:name => "activity[descriptions][]", :size => "50", :type => "text", :value => ""}/
-      = link_to "javascript:" do
-        %span{ class: "icon-trash" } &nbsp;
-
-
-= content_for :javascript do
```

Comment:
@linko, I added check to the js files. So, they would be called only in right time. I think put js code in view is a bad practice. Why i should do this?